### PR TITLE
About us

### PR DESCRIPTION
**Pull request by Kengaroo**
About Us page (merged with dev)

### CODE REVIEW

**Review by cruiser22 — APPROVED** [review]
ok

File: src/View/about/about.html.twig [hunk 1]
```diff
@@ -1,23 +1,27 @@
 {% extends 'layout.html.twig' %}
 	{% block title %} {{parent}} A propos
-			{% endblock %}
-
+    {% endblock %}
+    {% block stylesheet%}
+        <link rel="stylesheet" href="/assets/css/about.css">
+    {% endblock %}
 {% block content %}
 	<h1>Qui nous sommes </h1>
```
Comment (Benja-san) [inline src/View/about/about.html.twig]: please wrap everything inside a main tag

File: src/View/about/about.html.twig [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
-    <div class="container-about">	
-    <p>Salut !<br/>Nous sommes une équipe de 5 devs-voyagers. 
-        On apprecie le bon pain (et le vin) ainsi que les bon jeux du cirque. 
+    <div class="container-about">
+    <div class="text">
+    <h3>Salut !</h3>
```
Comment (Benja-san) [inline src/View/about/about.html.twig]: where is the h2 ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/View/about/about.html.twig [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-	{% block title %} {{parent}} A propos
+	{% block title %} {{ parent() }} - A propos
     {% endblock %}
     {% block stylesheet%}
         <link rel="stylesheet" href="/assets/css/about.css">
     {% endblock %}
 {% block content %}
-	<h1>Qui nous sommes </h1>
-    <div class="container-about">
+	<h1 class="about">Qui nous sommes </h1>
```
Comment (Benja-san) [inline src/View/about/about.html.twig]: h1 must be inside main tag

File: src/View/layout.html.twig [hunk 4]
```diff
@@ -23,18 +23,17 @@
 
 	</head>
 <body>
-
     {% block header %} 
-    {% include 'Components/_header.html.twig' %}
+    	{% include 'Components/_header.html.twig' %}
```
Comment (Benja-san) [inline src/View/layout.html.twig]: if your header and footer is available in all pages, you should put it ahead of header / footer block

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**